NEWDB/Mongo/WNR.js

```diff
@@ -0,0 +1,114 @@
+const faker = require('faker');
+const fs = require('fs');
+const v8 = require('v8');
+
+const fileContent = `
+  use reviews;
+  db.dropDatabase();
+  use reviews;
+
+  db.createCollection('products');
+  db.createCollection('users');
+  db.createCollection('reviews');
+
+
+
+`;
+
+
+const checkMemoryNative = () => {
+  console.log('Memory Usage: ', process.memoryUsage());
```

Comment:
what does this process.memoryUsage() do?

Reply:
this console logs how much memory is being used so you can see if it is getting close to crashing